mesonbuild/interpreterbase/decorators.py

```diff
@@ -631,8 +631,6 @@ def use(self, subproject: 'SubProject', location: T.Optional['mparser.BaseNode']
         feature_key = (self.feature_name, location)
         if feature_key in register[self.feature_version]:
             # Don't warn about the same feature multiple times
-            # FIXME: This is needed to prevent duplicate warnings, but also
-            # means we won't warn about a feature used in multiple places.
```

Comment:
Good catch, but the actual fix was in 7a033c78870e55b00d10a6499b40ad98b88a0509.

Reply:
e.g. the discussion at https://github.com/mesonbuild/meson/pull/9488#discussion_r739301985